app/assets/javascripts/discourse/tests/integration/components/d-editor-test.js

```diff
@@ -728,6 +728,100 @@ third line`
     assert.strictEqual(this.value, "red yellow blue");
   });
 
+  async function indentSelection(container, direction) {
```

Comment:
I tried to test this with selecting text and firing the key press event for TAB, but sadly it was not working, An app event seems to be much more reliable.